### ISSUE TITLE
No GPS Data with Android 8.1 on OnePlusOne 5T

### ISSUE THREAD

**Issue by Libero793** [BaseflowIT/flutter-geolocator#139]
## 🐛 Bug Report

No Data is returned under Android 8.1 on OnePlusOne 5t with Android 8.1
### Expected behavior
Return GPS Information on 

gps.Position position = await gps.Geolocator().getCurrentPosition(desiredAccuracy: gps.LocationAccuracy.high);

### Configuration

**Version:** 1.x

**Platform:** 
- [ ] :iphone: iOS
- [x ] :robot: Android

**Comment by ibrierley** [comment]
Have you tested it outside where you can get a signal ?
Also have you got the following in AndroidManifest ?

<uses-permission android:name="android.hardware.location.gps" />
<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />

**Comment by ronaldG19** [comment]
Nothing worked for me

**Comment by ibrierley** [comment]
Out of interest, is it the same if listening to a stream, does it ever
return anything ? I also assume it's being tested outside where you can get
a GPS signal and location is enabled.

**Comment by Libero793** [comment]
Yep tested and doesnt work - you just getting no response from the stream

**Comment by StijnWoerkom** [comment]
Could you add some reproduce steps so we can understand your situation?

**Comment by mvanbeusekom** [comment]
Closed due to no feedback
